ic_routing_board_generation/board_generator/board_generator_v3_1_0_rb.py

```diff
@@ -0,0 +1,476 @@
+# constants has been moved to jumanji/environments/combinatorial/routing/constants.py
+# import jumanji.environments.combinatorial.routing.constants
+#from jumanji.environments.combinatorial.routing.constants import TARGET, HEAD, EMPTY
+#from jumanji.environments.combinatorial.routing.constants import SOURCE as WIRE
+HEAD, TARGET, WIRE, EMPTY = 4,3,2,0
+# Also available to import from constants OBSTACLE, NOOP, LEFT, LEFT, UP, RIGHT, DOWN
+from dataclasses import dataclass
+import numpy as np
+from copy import deepcopy
+import random
+from typing import List, Tuple
+#from jax.numpy import asarray # Currently jaxlib is not supported on windows.  This will have to be sorted.
+#from env_viewer import RoutingViewer # Currently jaxlib is not supported on windows.  This will have to be sorted.
+
+@dataclass#
+class Position:
+    # Class of 2D tuple of ints, indicating the size of an array or a 2D position or vector.
+    x: int
+    y: int
+
+class Board:
```

Comment:
As before, please rename class

Reply:
you can also consider inheriting from a version and only changing the code that is different rather than reimplementing the whole thing - I did something similar for Ugo's boards if you want to see what I mean